modelling/src/main/java/org/axonframework/modelling/command/LockingRepository.java

```diff
@@ -281,6 +286,12 @@ public Builder<T> subtype(@Nonnull Class<? extends T> subtype) {
             return this;
         }
 
+        @Override
```

Comment:
Did you update the `builder` method and `Builder` class JavaDoc?

Reply:
You've missed adjusting the JavaDoc of the `static abstract class Builder` of this file.